source/agent/addons/quic/QuicTransportConnection.cc

```diff
@@ -116,9 +117,20 @@ NAUV_WORK_CB(QuicTransportConnection::onStreamCallback)
     }
 }
 
-NAN_METHOD(QuicTransportConnection::createBidirectionalStream){
+NAN_METHOD(QuicTransportConnection::createBidirectionalStream)
+{
     QuicTransportConnection* obj = Nan::ObjectWrap::Unwrap<QuicTransportConnection>(info.Holder());
-    auto stream=obj->m_session->CreateBidirectionalStream();
+    auto stream = obj->m_session->CreateBidirectionalStream();
     v8::Local<v8::Object> streamObject = QuicTransportStream::newInstance(stream);
     info.GetReturnValue().Set(streamObject);
+}
+
+NAN_METHOD(QuicTransportConnection::close)
```

Comment:
I don't know the usage of closeCode and reason here.
If all `closeCode` are 0, and the content of `reason` string does not matter, why not hide them in JS layer and just pass (0, '') as default.

Reply:
It just follows https://w3c.github.io/webtransport/#ref-for-dom-webtransport-close and https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3-02#section-5 to terminate a session. The CLOSE_WEBTRANSPORT_SESSION Capsule is introduced in the most recent version. Since we don't have a definition for the error code or error message at this time, it uses zero and empty string. Actually the `closeInfo` is an object in the WebTransport spec, it's split into two arguments here just for easy implementation. Perhaps it's better to follow the spec and combine them into an object and make it optional.

Reply:
Just checked the code, `closeInfo` is an object, but it's not optional in this implmenetation.

Reply:
You could check Info.Length is 0,

Reply:
Done.